<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hsys_widget</class>
 <widget class="QWidget" name="hsys_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_7">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_8">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_9">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton_sin">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButton_ln">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="pushButton_fan">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>1/x</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_5">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_cos">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_log">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>log</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pushButton_pinfang">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>x^2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_1">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_tan">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButton_pi">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Ï€</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="pushButton_zhimi">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>x^y</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_0">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_dengyu">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pushButton_quanqing">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>å…¨æ¸…</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButton_tuige">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>é€€æ ¼</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_zhengfu">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
